### ISSUE TITLE
Add a separate input field for requirements and separate it from the goal input field

### ISSUE THREAD

**Issue by Passerby1011** [reworkd/AgentGPT#128]
![image](https://user-images.githubusercontent.com/63802758/232088889-7a174646-55a1-49a3-aaa4-7ac3dabc7f0f.png)
"I think it would be helpful to separate the fields for goals and requirements. Perhaps add a requirements field below the goals field. In my experience, setting a goal always involves accompanying requirements, and there might be more than one requirement. Entering them all into one field could become confusing

**Comment by Alimjan2013** [comment]
i agree with this. actually when our goals are specific we need provide background information too
